test/load/deployment-template.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{.Name}}
+  labels:
+    group: {{.Group}}
+spec:
+  replicas: {{.Replicas}}
+  selector:
+    matchLabels:
+      name: {{.Name}}
+  template:
+    metadata:
+      labels:
+        name: {{.Name}}
+        group: {{.Group}}
+    spec:
+      containers:
+      - name: nginx
+        image: nginx
```

Comment:
unless you need nginx specifically, I would change this to `k8s.gcr.io/pause-amd64:3.1` or something that's in the VHD so we don't have to fetch the image